.travis/post_before_script.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+set -euv
+
+# Aliases for running commands in the pulp-worker container.
+PULP_API_POD="$(sudo kubectl get pods | grep -E -o "pulp-api-(\w+)-(\w+)")"
+PULP_CONTENT_APP_POD="$(sudo kubectl get pods | grep -E -o "pulp-content-(\w+)-(\w+)")"
```

Comment:
Try getting rid of the outer quotes.

We do not have them in before_script, the otuput will always be a simple alphanum with dashes string, and I think the inner quotes + outer quotes mean incorrect quoting.

Reply:
@mikedep333 to confirm you mean this?

```
diff --git a/.travis/post_before_script.sh b/.travis/post_before_script.sh
index d4dfd05..2acf7be 100755
--- a/.travis/post_before_script.sh
+++ b/.travis/post_before_script.sh
@@ -3,8 +3,8 @@
 set -euv
 
 # Aliases for running commands in the pulp-worker container.
-PULP_API_POD="$(sudo kubectl get pods | grep -E -o "pulp-api-(\w+)-(\w+)")"
-PULP_CONTENT_APP_POD="$(sudo kubectl get pods | grep -E -o "pulp-content-(\w+)-(\w+)")"
+PULP_API_POD=$(sudo kubectl get pods | grep -E -o "pulp-api-(\w+)-(\w+)")
+PULP_CONTENT_APP_POD=$(sudo kubectl get pods | grep -E -o "pulp-content-(\w+)-(\w+)")
 
 CMD_API_PREFIX="sudo kubectl exec -i $PULP_API_POD -- "
 CMD_CONTENT_APP_PREFIX="sudo kubectl exec -i $PULP_CONTENT_APP_POD -- "
```

Reply:
Yes

Reply:
The problem was actually due to there being 2 pulp-content pods. So the 2nd pulp-content pod name was passed as the command.

Assuming you are OK with there being 2 of them, modify the PULP_CONTENT `grep` command to include the arg:
-m 1

(It can be in any position in the list of args for grep.)